packages/service-integration/src/service_integration/cli/__init__.py

```diff
@@ -0,0 +1,72 @@
+# Allows entrypoint via python -m as well
+
+from typing import Annotated
+
+import rich
+import typer
+
+from .._meta import __version__
+from ..settings import AppSettings
+from . import _compose_spec, _metadata, _run_creator, _test
+from ._config import config_app
+
+app = typer.Typer()
+
+
+def _version_callback(value: bool):  # noqa: FBT002
+    if value:
+        rich.print(__version__)
+        raise typer.Exit
+
+
+@app.callback()
+def main(
+    ctx: typer.Context,
+    registry_name: Annotated[
+        str,
+        typer.Option(
+            "--REGISTRY_NAME",
+            help="image registry name. Full url or prefix used as prefix in an image name",
+        ),
+    ] = None,
+    compose_version: Annotated[
+        str,
+        typer.Option(
+            "--COMPOSE_VERSION",
+            help="version used for docker compose specification",
+        ),
+    ] = None,
+    version: Annotated[  # noqa: FBT002
+        bool,
+        typer.Option(
+            "--version",
+            callback=_version_callback,
+            is_eager=True,
+        ),
+    ] = False,
+):
+    """o2s2parc service Integration Library (OOIL in short)"""
+    assert isinstance(version, bool | None)  # nosec
+
+    overrides = {}
+    if registry_name:
+        overrides["REGISTRY_NAME"] = registry_name
+
+    if compose_version:
+        overrides["COMPOSE_VERSION"] = compose_version
+
+    # save states
+    ctx.settings = AppSettings.parse_obj(overrides)
+
+
+#
+# REGISTER commands and/or sub-apps
+#
+
+app.command("compose")(_compose_spec.create_compose)
+app.add_typer(config_app, name="config", help="Manage osparc config files")
```

Comment:
Minor: I don't see this helper in the screenshot of the command line interface. Shouldn't it be there?

Maybe the screenshot is not up-to-date or I am misunderstanding. 😊 

![image](https://github.com/ITISFoundation/osparc-simcore/assets/18575092/a5d59b5b-1e67-4f0e-8cd0-7f28c703cf99)

Reply:
old version 
![image](https://github.com/ITISFoundation/osparc-simcore/assets/32402063/d88e9e6f-7a2e-4640-bc05-26e94fc71140)